doc/python/selections.md

```diff
@@ -0,0 +1,139 @@
+---
+jupyter:
+  jupytext:
+    notebook_metadata_filter: all
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.7
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+  language_info:
+    codemirror_mode:
+      name: ipython
+      version: 3
+    file_extension: .py
+    mimetype: text/x-python
+    name: python
+    nbconvert_exporter: python
+    pygments_lexer: ipython3
+    version: 3.9.0
+  plotly:
+    description: How to use selections in Python. Examples of adding and styling selections.
+    display_as: file_settings
+    language: python
+    layout: base
+    name: Selections
+    order: 26
+    permalink: python/selections/
+    thumbnail: thumbnail/make_selection.jpg
+---
+
+## Adding Selections
+
+*New in 5.10*
+
+You can add persistent selections to a rendered figure using the **Box Select** and **Lasso Select** tools in the mode bar.
+To add multiple selections, select **Shift** when making new selections.
+To clear a selection, double-click it. On a subplot you can clear all selections by double-clicking any unselected area of the subplot.
+
+
+
+You can also add selections to a figure that displays when it renders using `fig.add_selection`.
+Here, we add a rectangular selection with a region between `3.0` and `6.5` on the x axis and between `3.5` and `5.5` on the y axis.
+
+
+```python
+import plotly.express as px
+
+df = px.data.iris()
+
+fig = px.scatter(df, x="sepal_width", y="sepal_length")
+fig.add_selection(type="rect", x0=3.0, y0=6.5, x1=3.5, y1=5.5)
+
+fig.show()
+```
+
+## Selections using Path
+
+
+In the above example, we added a rectangular selection (`type="rect"`). You can also render a custom SVG for a selection by setting `type="path"` and defining a `path`. Here, we create a selection with the path "M2,6.5L4,7.5L4,6Z"
+
+```python
+import plotly.express as px
+
+df = px.data.iris()
+
+fig = px.scatter(df, x="sepal_width", y="sepal_length")
+fig.add_selection(type="path", path="M2,6.5L4,7.5L4,6Z")
+
+fig.show()
+```
+
```

Comment:
I suggest we also provide an example using `date` axis to illustrate how to position selection regions.
Something like https://github.com/plotly/plotly.js/commit/1107600f7a9be2c11cd31c800bd46675d1545acf

Reply:
I've added an example.